doc/source/whatsnew/v0.22.0.txt

```diff
@@ -17,41 +17,157 @@ New features
 -
 -
 
+
+.. _whatsnew_0210.enhancements.get_dummies_dtype:
+
+``get_dummies`` now supports ``dtype`` argument
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :func:`get_dummies` now accepts a ``dtype`` argument, which specifies a dtype for the new columns. The default remains uint8. (:issue:`18330`)
+
+.. ipython:: python
+
+   df = pd.DataFrame({'a': [1, 2], 'b': [3, 4], 'c': [5, 6]})
+   pd.get_dummies(df, columns=['c']).dtypes
+   pd.get_dummies(df, columns=['c'], dtype=bool).dtypes
+
+
 .. _whatsnew_0220.enhancements.other:
 
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
-- Better support for ``Dataframe.style.to_excel()`` output with the ``xlsxwriter`` engine. (:issue:`16149`)
--
--
+``Series.rank`` ``DataFrame.rank`` now can handle ``inf`` values and missing values properly
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+In previous versions, ``inf`` elements were assigned ``NaN`` as their ranks. Now ranks are calculated properly. (:issue:`6945`)
+
+Previous Behavior
+
+.. code-block:: ipython
+
+    In [17]: pd.Series([-np.inf, 0, 1, np.inf]).rank()
+    Out[17]:
+    0    1.0
+    1    2.0
+    2    3.0
+    3    NaN
+
+Current Behavior
+
+.. ipython:: ipython
+
+    In [5]:  pd.Series([-np.inf, 0, 1, np.inf]).rank()
+    Out[5]:
+    0    1.0
+    1    2.0
+    2    3.0
+    3    4.0
+    dtype: float64
+
+Furthermore, in previous versions, missing values were not distinguished from infinit values in the calculation.
+
+Previous Behavior
+
+.. code-block:: ipython
+
+    In [15]: pd.Series([np.nan, np.nan, -np.inf, -np.inf]).rank(na_option='top')
```

Comment:
would think about combining this with the above example (ok if you can't / better like this)